update/stock_update.js

```diff
@@ -0,0 +1,122 @@
+//---CONFIGURE AUTHENTICATION HERE--------------------
+var CLOUDANT_KEY      = 'XXXXXXXXX';
```

Comment:
this is fine for first pass but it should ideally be read from a file, like here: https://github.com/IBM/watson-review-analyzer/blob/master/app.js#L15-L26

```
config = jsonfile.readFileSync( __dirname + '/config.json' );

app.use(function (req, res, next){
  req.config = config;
  next();
});

var discovery = new DiscoveryV1({
  username: config.discoveryUsername,
  password: config.discoveryPassword,
  version_date: '2017-11-07'
});
```

Reply:
you'll want to include arguments for disco and cloudant

Reply:
Yeah, thanks. The plan was to move to environment vars but that'll work too.

Reply:
That was the "user" could config them in the bluemix console and not have to edit any code.

Reply:
it's entirely do-able to do both, this way you can satisfy a local test run and a more production style run. Check env vars first, if they are there, use em. If not, check config file, else, bomb out.